api/apps/api/test/access-control/scenario-acl/scenario-acl-get.e2e-spec.ts

```diff
@@ -0,0 +1,59 @@
+import { FixtureType } from '@marxan/utils/tests/fixture-type';
+import { getFixtures } from './scenario-acl.fixtures';
+
+let fixtures: FixtureType<typeof getFixtures>;
+
+beforeEach(async () => {
+  fixtures = await getFixtures();
+});
+afterEach(async () => {
+  await fixtures?.cleanup();
+});
+
+test(`getting scenario users when user has no role in the scenario`, async () => {
+  const scenarioId = await fixtures.GivenScenarioWasCreated();
+  const response = await fixtures.WhenGettingScenarioUsersAsNotInScenario(
+    scenarioId,
+  );
+  fixtures.ThenForbiddenIsReturned(response);
```

Comment:
Consider creating reusable step
```
fixtures.WhenGettingScenarioUsers(scenarioId, userId)
```

Reply:
The one that is reused in other tests of the other endpoints is `WhenGettingScenarioUsersAsOwner` as it is needed to check that a scenario has been updated with new users and also to check that the users you deleted are effectively gone.

I didn't want to use something generic like `WhenGettingScenarioUsers` because this action is defined by the user doing the action and a general one can take away some essential information that needs to be coupled together with the action itself.